pycheribuild/projects/cross/cheribsd.py

```diff
@@ -1239,15 +1593,18 @@ def setup_config_options(cls, install_directory_help=None, **kwargs):
                                                  help="Path to an MFS root image to be embedded in the kernel for "
                                                       "booting")
 
+        cls.default_kernel = cls.add_config_option("default-kernel", show_help=True, _allow_unknown_targets=True,
```

Comment:
Maybe this should be "default-kernel-abi" instead? Might make the --help confusing otherwise.

Reply:
And I guess it also wants `only_add_for_targets`?